setup_ml.py

```diff
@@ -81,7 +81,7 @@ def find_ml_packages(root):
         "install": install_plat
     },
     zip_safe=False,
-    install_requires=['vineyard', 'tensorflow', 'xgboost', 'torch', 'mxnet'],
+    install_requires=['vineyard', 'tensorflow', 'xgboost', 'torch', 'mxnet', 'nvidia-dali-cuda110'],
```

Comment:
I think we need to add a profile, namely `gpu`, where the `nvidia-dali-cuda110` is listed as a dependency, as it is not available on GPU.

Reply:
Yes dali requires cuda and nvidia drivers. I am not sure how we can add it here

Reply:
Its all right, I can assistant to revise the PR.

Reply:
Thank You